.drone.yml

```diff
@@ -0,0 +1,36 @@
+pipeline:
+
+  deploy_to_notprod:
+    image: quay.io/ukhomeofficedigital/kd
+    environment:
+      - KUBE_NAMESPACE=dq-apps-notprod
+      - INSECURE_SKIP_TLS_VERIFY=true
+      - URL=analysis.staging.notprod.dq.homeoffice.gov.uk
+    commands:
+      - export KUBE_TOKEN=$$NOTPROD_KUBE_TOKEN
+      - export KUBE_SERVER=$$NOTPROD_KUBE_SERVER
+      - kd -f network-policy.yaml -f service.yaml -f ingress.yaml -f deployment.yaml
+    secrets:
+      - NOTPROD_KUBE_TOKEN
+      - NOTPROD_KUBE_SERVER
+    when:
+      event: push
+      branch: master
```

Comment:
Remove line 18 so that commits are deployed to NotProd

Reply:
ok